CCDFDataModel/CTime.cpp

```diff
@@ -972,9 +987,12 @@ double CTime::quantizeTimeToISO8601(double offsetOrig, CT::string period, CT::st
       hmsPart.replaceSelf("D", "");
       int D = hmsPart.toInt();
       int origD = date.day;
-      date.day = origD - origD % D;
+      int restD = origD % D;
```

Comment:
I don't see how this would make the code significantly clearer, so I don't feel it is necessary to make use a function here. It will probably not even save lines in the code.